### ISSUE TITLE
Time field not updated on touch devices

### ISSUE THREAD

**Issue by rnissan** [jonthornton/jquery-timepicker#441]
When selecting a time from the dropdown on a touch device, the time in the drop is highlighted but the time field is not updated. There appears to be a missing 'touchstart' event handler.

I changed Line 566 on jquery.timepicker.js from  

`list.on('mousedown', 'li', function(e) {`

to 

`list.on('touchstart mousedown', 'li', function(e) {`

and it seems to have fixed the problem.

**Comment by jonthornton** [comment]
I wasn't able to reproduce this with Safari on an iPhone 6. Which device/browser were you using?

**Comment by rnissan** [comment]
I had the problem on an iPhone 6 plus on iOS 9, iPad air 2 on iOS 9 and an older iPad on iOS 8. both safari and chrome (which i think both use the same webkit engine)

**Comment by jonthornton** [comment]
Binding to `touchstart` causes the time to update any time you scroll the dropdown on a touch device. The current behavior is correct.

The time highlight updates due to the hover styling. You can disable this with CSS.

**Comment by wootencl** [comment]
Thanks for the excellent timepicker @jonthornton . I also seem to be having the same problem of the dropdown not closing on iOS mobile devices. I first tried it on xcode simulator and then verified it with my iPhone 6 (iOS 9.3). Messing around with it on my iPhone it seems to register a select only after holding a dropdown option for a significant amount of time (to the point the cut/copy options pop up). Getting late where I'm located and I haven't had a chance to peak at the source JS but plan on doing so tomorrow to see if I can see if there's a quick fix. Until then I figured I'd bring it back to you attention
